Corzbank.Data/CorzbankDbContext.cs

```diff
@@ -17,7 +17,47 @@ public CorzbankDbContext(DbContextOptions<CorzbankDbContext> options)
         {
         }
 
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<DepositCard>().HasKey(x => new { x.CardId, x.DepositId });
```

Comment:
let's create a rule for consistence. Name lambda key as a shortcut to the class you are referred to
**Ex**: DepositCard (dc) => modelBuilder.Entity<DepositCard>().HasKey(dc => new { dc.CardId, dc.DepositId });

Reply:
Refactor in other places as well